frontend/src/pages/projects/pvc/MountPathField.tsx

```diff
@@ -37,8 +37,8 @@ const MountPathField: React.FC<MountPathFieldProps> = ({
           onChange={(e, value) => {
             let error = '';
             if (value.length === 0) {
-              error = 'Required';
-            } else if (!/^[a-z-]+$/.test(value)) {
+              error = 'Enter a path to a model or folder. This path cannot point to a root folder.';
```

Comment:
Was this text been provided by UX? Maybe @kaedward could verify this tech content.

Reply:
Yeah I just moved the old text here https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/projects/pvc/MountPathField.tsx#L55 to above because the helper text render logic was incorrect before.